### PR TITLE
Added rlite-router@1.0.6

### PR DESCRIPTION
**Pull request by chrisdavies**
Adding my JavaScript routing library. Thanks!

### CODE REVIEW

File: ajax/libs/rlite-router/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "npmName": "rlite-router",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "/var/folders/b4/h23tzdm93l563b9ly_5gkyd80000gn/T/tmp-59949w8WBtVhSMvnn/README.md"
```
Comment (IonicaBizau) [inline ajax/libs/rlite-router/package.json]: Hmm, did you use `cdnjs-importer` for that? This looks like a bug in cdnjs-importer.

**Comment by PeterDaveHello** [comment]
Will people use test lib on cdn?

File: ajax/libs/rlite-router/package.json (hunk 1 above)
Comment (chrisdavies) [inline ajax/libs/rlite-router/package.json]: Yeah. I did... I'll resubmit sometime soon. Sorry!

**Comment by chrisdavies** [comment]
Nope. They'd use rlite.min.js, and the plugin. I'll make a manual pull request in a bit.

File: ajax/libs/rlite-router/package.json (hunk 1 above)
Comment (IonicaBizau) [inline ajax/libs/rlite-router/package.json]: I will investigate what happened. *I am* sorry!

Comment (IonicaBizau) [inline ajax/libs/rlite-router/package.json]: I reproduced this issue on my machine and opened an issue: https://github.com/IonicaBizau/cdnjs-importer/issues/10

In the meantime you can manually change the filename so we can merge this.

**Pull request closed without merging**